build/wix/subdirs.txt

```diff
@@ -0,0 +1,7 @@
+controllers
```

Comment:
Of course !

Even Better, I have plans to remove that file completely and also generate them with a for loop in the (temporary) demo batch files.

Didn't I push this already ?